content/en/user-guide/ci/circle-ci/index.md

```diff
@@ -1,45 +1,118 @@
+- You'll be prompted to add a configuration. You can add the configuration manually or choose a starter pipeline.
+- If you choose the starter CI pipeline, a sample `config.yml` file is created and committed to a `circleci-project-setup` branch in your repo.
+- You can add the following configuration to the `config.yml` file to start LocalStack and run your tests against it.
+
+{{< tabpane >}}
+{{< tab header="Community" lang="yml" >}}
 version: 2.1
+
 orbs:
-  localstack: localstack/platform@1.0
+  python: circleci/python@2.0.3
+
 jobs:
-  run-integration-tests:
-    executor: localstack/default
+  example-job:
+    machine:
+      image: ubuntu-2004:2022.04.1
+
     steps:
-      - localstack/startup
+      - checkout
+
       - run:
-          command: awslocal s3 mb s3://test-bucket
+          name: Start LocalStack
+          command: |
+            pip3 install localstack awscli-local[ver1]
+            docker pull localstack/localstack
+            localstack start -d                     
+
+            echo "Waiting for LocalStack startup..."  
+            localstack wait -t 30                     
+            echo "Startup complete"
+            
       - run:
-          command: echo "Execute your tests here :)"
+          name: Run AWS CLI commands against LocalStack
+          command: |
+            awslocal s3 mb s3://test-bucket
+            awslocal sqs create-queue --queue-name test-queue
+            awslocal sns create-topic --name test-topic
+
 workflows:
-  integration-test:
+  version: 2
+  build:
     jobs:
-      - run-integration-tests
-```
+      - example-job
+{{< /tab >}}
+{{< tab header="Pro" lang="yml" >}}
+version: 2.1
+
+orbs:
+  python: circleci/python@2.0.3
+
+jobs:
+  example-job:
+    machine:
+      image: ubuntu-2004:2022.04.1
+    environment:
+      LOCALSTACK_API_KEY: YOUR_API_KEY_HERE
+
+    steps:
+      - checkout
+
+      - run:
+          name: Start LocalStack
+          command: |
+            pip3 install localstack awscli-local[ver1]
+            docker pull localstack/localstack-pro
+            localstack start -d
+            echo "Waiting for LocalStack startup..."
+            localstack wait -t 30
+            echo "Startup complete"
+            
+      - run:
+          name: Run AWS CLI commands against LocalStack
+          command: |
+            awslocal s3 mb s3://test-bucket
+            awslocal sqs create-queue --queue-name test-queue
+            awslocal sns create-topic --name test-topic
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - example-job
+{{< /tab >}}
+{{< /tabpane >}}
+
+The above CircleCI job does the following:
+
+- Defines a job called `example-job` that installs the `localstack` CLI and `awslocal` wrapper script to execute AWS CLI commands against LocalStack.
+- Pulls the LocalStack Docker image depending on the product tier (For Community, the image is `localstack/localstack`, while for Pro it is `localstack/localstack-pro`).
```

Comment:
```suggestion
- Pulls the LocalStack Docker image depending on the product tier (For Community, the image is `localstack/localstack`, while for Pro+ it is `localstack/localstack-pro`).
```

I guess over time we need a good strategy to collectively refer to the set of commercial tiers (Pro (future: Starter) / Team / Enterprise). We can tackle that across our docs one we roll out the layout of the new product tiers.. 👍